File: TwoSum.java
import java.util.Arrays;

public class TwoSum {
    public int[] twoSum(int[] nums, int target) {
        Arrays.sort(nums);
        int i = 0, j = nums.length - 1;
        while (i < j) {
            if (nums[i] + nums[j] == target) {
                return new int[] { i, j };
            } else if (nums[i] + nums[j] < target) {
                i++;
            } else {
                j--;
            }
        }
        return new int[] { 0, 0 };
    }
}

File: Twosum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        nums.sort()
        i=0
        j=len(nums)-1
        while i < j:
            if nums[i]+nums[j] == target:
                return [i,j]
            elif nums[i]+nums[j] < target:
                i+=1
            else:
                j-=1
        return [-1,-1]
        